src/dhcp_device.cpp

```diff
@@ -410,6 +433,14 @@ static void handle_dhcp_option_53(std::string &sock_if,
             packet_valid_type = DHCP_VALID;
         }
         break;
+
+    case MALFORMED:
+        // The packet need to be dropped, e.g., checksum error
+        syslog(LOG_WARNING, "handle_dhcp_option_53(%s): Drop packet found, option 53 type %d", context->intf, dhcp_option[2]);
+        
+        // The packet type need to set as unknown as we still want to count it.
+        packet_valid_type = DHCP_UNKNOWN;
```

Comment:
We break here, then in line 454, the packet is counted as original type witch this func?
 increase_cache_counter_per_interface(sock_if, context, dhcp_option[2], dir);

Also, why there is packet with dhcp_option[2] set to MALFORMED?